tests/TenantDatabaseManagerTest.php

```diff
@@ -100,20 +100,22 @@
 
     $tenant = Tenant::create();
 
-    expect(array_keys(app('db')->getConnections()))->toBe(['central']);
+    expect(array_keys(app('db')->getConnections()))->not()->toContain('tenant');
```

Comment:
Can you explain this change?

Reply:
And the changes below as well — using `contain` instead of strict `toBe()` assertions.

Reply:
> Can you explain this change?

So, We added the host manager in this PR, which internally creates the `tenant_host` connection. 

As We are creating tenant-right above line # 101. Tenant creation creates a host connection already. So now there are two connections in the array. 
<img width="276" alt="image" src="https://user-images.githubusercontent.com/54532330/191902425-ebeafc5c-61ff-4d46-b716-0d8cde83aebf.png">

That's why I tweaked the assertions a little bit. Now assertions are specific to the tenant. I mean test says `the tenant connection is fully removed`, so assertions are checking if the tenant connection exists, not exists, etc. 

> using contain instead of strict toBe() assertions.

We can use toBe(), but we will be checking two connections (as in the above image), so I think checking contains is good because we are ensuring the tenant connection exists.

Reply:
Is the point of the `toContain()` assertion to check that it's `tenant` or `tenant_host_connection`? If so, I'd definitely change that to be more strict.

The name of `tenant_host_connection` is customizable, plus connection simply having "tenant" in it doesn't necessarily mean anything.

So I'd just make the assertions more strict.

Reply:
> So I'd just make the assertions more strict.

Sure. But I don't see any benefit of more strict assertions here. We were testing these points in this test. 

- assert `tenant` connection does not exist.
- initialize tenancy  - asserts that a `tenant` connection is created 
- end tenancy - assert `tenant` connection got removed. 

And the current assertions are doing the same. Does the test do anything differently? If so, please explain we will change the assertions. 

> plus connection simply having "tenant" in it doesn't necessarily mean anything.

It does mean that the connections array has a `tenant` connection along with other connections.

Reply:
> But I don't see any benefit of more strict assertions here

It's what I wrote here:

> The name of `tenant_host_connection` is customizable, plus connection simply having "tenant" in it doesn't necessarily mean anything.

We want to assert **actual checks that the connection is a certain value**, not that it contains a substring.

> It does mean that the connections array has a tenant connection along with other connections.

How? If I create a central connection and name it `tenants` (because it stores the list of tenants), then that's a central connection that passes your `toContain()` check.

Just make direct assertions about what the values equal. Don't make indirect assertions about the values containing some substring.

Reply:
>  Just make direct assertions about what the values equal. Don't make indirect assertions about the values containing some substring.

Sure.